experimental/casemapping/tests/gen_greek_to_me.rs

```diff
@@ -133,6 +133,12 @@ pub(crate) const GREEK_DATA_TRIE: CodePointTrie<'static, GreekPrecomposedLetterD
 {output}
 ========================================================"#
         );
+        // For whatever reason; this fails to work properly as a golden test on windows CI
+        // even though we pass --newline_style.
+        //
+        // We can still have it print the output but require folks pass --nocapture
+        // if they wish to regen on Windows, and let the test itself pass.
+        #[cfg(not(windows))]
```

Comment:
@eggrobin mind removing this line and checking if `cargo test --all-features --test gen_greek_to_me` fails for you (and if so, what the actual diff is)

Reply:
It fails for me. Copying the output results in no diff.

Removing the line
```rs
        .arg("newline_style=unix")
```
fixes the test (the file probably gets checked out with the local style of line termination).

Reply:
oh goddamnit it's the git thing

this is dependent on local git settings

lemme rewrite the test.